test/test_tailer.rb

```diff
@@ -0,0 +1,98 @@
+require 'mongoriver'
+require 'mongo'
+require 'minitest/autorun'
+require 'mocha/setup'
+
+describe 'Mongoriver::Tailer' do
+  class CursorStub
+    def initialize
+      @events = []
+      @current = 0
+    end
+
+    def add_option(opt) end
+
+    def generate_ops(max)
+      @current = 0
+      (1..max).map do |id|
+        @events << {
+          'ts' => BSON::Timestamp.new(Time.now.to_i, 0),
+          'ns' => 'foo.bar',
+          'op' => 'i',
+          'o'  => {
+            '_id' => id.to_s
+          }
+        }
+      end
+    end
+
+    def has_next?
+      @current < @events.length
```

Comment:
I think current is 0 indexed and length is not right. Wouldn't this be an off by one error?